factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/utils/filterTitleFromJson.kt

```diff
@@ -0,0 +1,14 @@
+package org.factcast.schema.registry.cli.utils
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+
+fun filterTitleFromJson(input: JsonNode): JsonNode {
```

Comment:
very minor style thing: i try to not repeat the type, like filterTitleFrom(JsonNode)...

Reply:
renamed